src/resources/js/ui_online.js

```diff
@@ -343,8 +344,32 @@ export function setUpUI() {
     const roomId = generatePushID();
     createRoom(roomId).then(() => {
       printCurrentRoomID(roomId);
+      copyBtn.style.visibility = 'visible';
     });
   });
+
+  copyBtn.addEventListener('click', () => {
+    const prettyRoomID = document.getElementById('current-room-id').textContent;
+    const language = getCurruntLanguage();
+
+    copyBtn.innerText = language == 'en' ? 'Copied!' : '복사됨!';
+    setTimeout(() => {
+      copyBtn.innerText = language == 'en' ? 'Copy' : '복사';
+    }, 500);
+
+    navigator.clipboard.writeText(prettyRoomID);
+  });
+
+  const getCurruntLanguage = () => {
```

Comment:
No i can fix it. i don't know about that. good!

Reply:
@gorisanson that's message just log literally.
How do you get language settings?
i want apply this more dynamic

Reply:
After adding hidden elements each of which contains "copy", "copied!" in `en/index.html` and "복사", "복사됨!" in `ko/index.html`,

so that "copy", "복사" have the same element ID, 
and "copied!", "복사됨!" have the same element ID,

we can use `getElementByID.textContent` to get the text.

Then, if a user is on English page, English text will be shown, if a user on Korean page, Korean text shown.

This is the method I'm using in this project :)

Reply:
@gorisanson 
How about another way like below?

```js
console.log('URL ', window.location.href.split('/')[3]); // ko or en
```

Reply:
I don't want hard coding the text on JS file.
I know this is my way so if you don't get it, I'll do it :)

Reply:
Ok i'll try that

Reply:
Ok. please show me the direction you think.

I'll follow you in this matter.

Reply:
Resolved by 1757bfd58ad17348fb90f8b23e28067e82989f2d.